File: frechman/service-1-java/src/main/java/ru/gavrilov/app/Application.java
package ru.gavrilov.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author gavrilov-sv
 * created on 07.06.2020
 */
@SpringBootApplication
@RestController
public class Application {

    @GetMapping("/")
    public String getWeString() {
        return "Мы";
    }

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: ann.kont/task2/src/main/kotlin/jug/ru/Application.kt
package jug.ru

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import java.net.URL

@SpringBootApplication
@RestController
open class Application {
  companion object {
    const val RESPONSE = "Мы"
    const val URL = "http://localhost:8080/"
  }

  @GetMapping("/")
  fun get(): String {
    return getResponse() + " сломали"
  }

  private fun doRequest(): String {
    return URL(URL).readText()
  }

  private fun getResponse(): String {
    val response = doRequest()
    if (RESPONSE != response) {
      throw Exception("Response is not: $RESPONSE it is: $response")
    }
    return response
  }
}

fun main(args: Array<String>) {
  SpringApplication.run(Application::class.java, *args)
}